composer.json

```diff
@@ -14,7 +14,7 @@
         "content management framework"
     ],
     "homepage": "https://github.com/laravel-backpack/CRUD",
-    "license": "MIT",
+    "license": "proprietary",
```

Comment:
No it's not!

@tabacitu - what should this be?

BTW this hasn't anything to do with German translations.

Reply:
yup - there was an issue somewhere on some package (can't remember), where someone complained that when typing ```composer licenses``` the result wasn't correct. And he was right. This change was to make sure people don't think it's MIT, when it's not. "proprietary" seemed appropriate because YuMmY is not that popular a license - composer doesn't yet have an option for it :-)

This is a welcome change, doesn't really matter if it was made here or elsewhere. I say we keep it, that's how composer.json should look anyway.